### ISSUE TITLE
MODULE_NOT_FOUND sync.js

### ISSUE THREAD

**Issue by diamkil** [mattwebbio/orbital-sync#188]
### What happened?

I installed Node.JS 18 and installed orbital-sync using the npm command. Upon trying to run it I get a module not found error..

### Version

1.5.2

### Log output

```shell
`node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/usr/local/lib/node_modules/orbital-sync/dist/sync.js' imported from /usr/local/lib/node_modules/orbital-sync/dist/index.js
    at new NodeError (node:internal/errors:405:5)
    at finalizeResolution (node:internal/modules/esm/resolve:324:11)
    at moduleResolve (node:internal/modules/esm/resolve:943:10)
    at defaultResolve (node:internal/modules/esm/resolve:1129:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:835:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
    at link (node:internal/modules/esm/module_job:76:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}`
```

**Comment by Eyal51** [comment]
Same for me. Not sure what's next.

**Comment by Oak-Trees** [comment]
Same here!

**Comment by mattwebbio** [comment]
Hey everyone, so sorry that it's taken me so long to get around to this! I just fixed the NPM release in v1.8.1. The following now works on my end:
```
npm install -g orbital-sync

export PRIMARY_HOST_BASE_URL=http://localhost:1111
export PRIMARY_HOST_PASSWORD=foobar 
export SECONDARY_HOSTS_1_BASE_URL=http://localhost:1112
export SECONDARY_HOSTS_1_PASSWORD=foobar

orbital-sync
```
